3-extra/1-radio-stations.js

```diff
@@ -26,6 +34,23 @@
  */
 // `getStations` goes here
 
+
+function getStations() {
```

Comment:
This looks perfect!
For practice, you could try re-writing this with the `filter` array method.